packages/freezed/lib/src/freezed_generator.dart

```diff
@@ -182,7 +185,10 @@ class FreezedGenerator extends ParserGenerator<_GlobalData, Data, Freezed> {
   }
 
   @override
-  Data parseElement(_GlobalData globalData, Element rawElement) {
+  Data parseElement(_GlobalData globalData, Element rawElement, {bool init}) {
```

Comment:
I don't quite understand what that init flag does. Could you explain?

Reply:
`parseElement()` gets called recursively through `_parseConstructorsNeedsGeneration()` and `_computeElementDataFor()`. 
But I only want to clear this `_parsedElementCheckList`, when its called by `GeneratorForAnnotation.generate()`,

Reply:
Why clear the list to begin with?